atc/db/team.go

```diff
@@ -732,6 +734,54 @@ func (t *team) OrderPipelines(names []string) error {
 	return nil
 }
 
+func (t *team) OrderPipelinesWithinGroup(groupName string, instanceVars []atc.InstanceVars) error {
```

Comment:
I guess this is more of a general question about ordering in general, but I was wondering what happens if you only order a few instances within a pipeline? For example, if I have 3 instances within a pipeline
```
instance1 -> secondary_ordering=1
instance2 -> secondary_ordering=2
instance3 -> secondary_ordering=3
```

And I run `fly -t <target> order-instance-pipelines -g <group> -p instance3 -p instance2`, it will set the `secondary_ordering` for `instance2` and `instance3`.
```
instance1 -> secondary_ordering=1
instance2 -> secondary_ordering=2
instance3 -> secondary_ordering=1
```

It will result in both `instance1` and `instance3` having `secondary_ordering` of 1, and in the UI I would see the ordering to be instance1 -> instance3 -> instance2. Do we want to just ignore the ordering of instances that are not passed into the fly command ordering? I know this is also a problem with the regular `fly order-pipelines` so it might not be necessary to address this here.

Reply:
My opinion of it is that the `order-[instance-]pipelines` command allows you to specify the relative ordering between pipelines you care about. If you don't include it in the list, then you probably don't care about where it actually appears.

But then this is also my opinion after reading the docs, this could be a case of "it's not a bug, it's a feature!"

Reply:
Exactly

Reply:
Oh ya I see that comment in the doc haha Does seem weird to keep it like that but honestly its not a big deal 😄